packages/atomic/src/components/facets-v1/facet-search/facet-search-matches.tsx

```diff
@@ -0,0 +1,35 @@
+import {FunctionalComponent, h} from '@stencil/core';
+import {i18n} from 'i18next';
+
+interface FacetSearchMatchesProps {
+  i18n: i18n;
+  query: string;
+  numberOfMatches: number;
+  hasMoreMatches: boolean;
+}
+
+export const FacetSearchMatches: FunctionalComponent<FacetSearchMatchesProps> = (
+  props
+) => {
+  if (!props.numberOfMatches) {
+    return (
+      <div class="ellipsed p-3 bg-neutral-light text-neutral-dark text-sm">
```

Comment:
![Screen Shot 2021-06-04 at 4 46 29 PM](https://user-images.githubusercontent.com/4923043/120860904-6b1e3a80-c554-11eb-92d9-10aff82f2c47.png)

The mockups have quotes around the query, but I haven't implemented it here because with the ellipsis it would be hackish to add the quote at the end it's naturally ellipsed with the rest. I'll ask Johny if we can use bold instead since it's what we do elsewhere for highlight